plugins/generator-1.16.5/forge-1.16.5/procedures/entity_set_oxygen.java.ftl

```diff
@@ -0,0 +1 @@
+if(${input$entity} instanceof LivingEntity)((LivingEntity)${input$entity}).setAir((int)${input$air});
```

Comment:
Set air is implemented for Entity so no need for instanceof check, check other procedures too if this applies to them.